src/main/java/uk/gov/hmcts/ccd/domain/service/listevents/DefaultListEventsOperation.java

```diff
@@ -6,29 +6,27 @@
 import uk.gov.hmcts.ccd.data.casedetails.CaseAuditEventRepository;
 import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
 import uk.gov.hmcts.ccd.domain.model.std.AuditEvent;
-
-import java.util.List;
 import uk.gov.hmcts.ccd.domain.service.common.UIDService;
 import uk.gov.hmcts.ccd.domain.service.getcase.CreatorGetCaseOperation;
 import uk.gov.hmcts.ccd.domain.service.getcase.GetCaseOperation;
 import uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException;
 import uk.gov.hmcts.ccd.endpoint.exceptions.ResourceNotFoundException;
 
+import java.util.List;
+
 @Service
 @Qualifier("default")
 public class DefaultListEventsOperation implements ListEventsOperation {
 
     private final CaseAuditEventRepository auditEventRepository;
     private final GetCaseOperation getCaseOperation;
     private final UIDService uidService;
-    public static final String RESOURCE_NOT_FOUND //
+    private static final String RESOURCE_NOT_FOUND //
         = "No case found ( jurisdiction = '%s', case type id = '%s', case reference = '%s' )";
+    private static final String CASE_EVENT_NOT_FOUND = "Case event not found";
 
     @Autowired
-    public DefaultListEventsOperation(CaseAuditEventRepository auditEventRepository,
-                                      @Qualifier(CreatorGetCaseOperation.QUALIFIER) final GetCaseOperation getCaseOperation,
-                                      UIDService uidService) {
-
+    public DefaultListEventsOperation(CaseAuditEventRepository auditEventRepository, @Qualifier(CreatorGetCaseOperation.QUALIFIER) final GetCaseOperation getCaseOperation, UIDService uidService) {
```

Comment:
this will break checkstyle because line length is above 120 I think

Reply:
yes its 120, changed my code style setting to 120 now.